debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordStateConfigDefinition.java

```diff
@@ -121,7 +121,7 @@ public static DeleteTombstoneHandling parse(String value, String defaultValue) {
     }
 
     public static final Field HANDLE_TOMBSTONE_DELETES = Field.create("delete.tombstone.handling.mode")
-            .withDisplayName("Handle delete records")
+            .withDisplayName("Handle delete and tombstone records")
```

Comment:
One could argue that a tombstone in Kafka is in fact a delete message. Thus `tombstone == delete`. And as we were using this in the UI as label for the field maybe we can leave the shorter name as it was before?

Reply:
Thanks for the review @rk3rn3r 
Although "tombstone" and "delete" are often used interchangeably, the connector can generate two distinct records: a tombstone record and a non-tombstone delete record with `__deleted: true`.

Since the new config `delete.tombstone.handling.mode` handles both types of records, I believe using a more descriptive name is appropriate.

In the UI, both `delete.tombstone.handling.mode` and `delete.handling.mode` currently have the same display name, which could cause confusion. Renaming the new config to something more specific would help clarify the difference. WDYT?

Reply:
I agree wit @rajdangwal on thise one. While tombstone is delete record in the plain Kafka context, in case of Debezium context these are two distinct records and it makes sense to differentiate between them. In fact tomstone is only technical one to enable compaction in Kafka, otherwise it would be completely unnecessary for Debezium.

Reply:
@rk3rn3r are we referring to change the config name from `delete.tombstone.handling.mode` to something else?